sql/src/test/java/io/crate/integrationtests/DiskUsagesITest.java

```diff
@@ -293,14 +294,18 @@ public void testMovesShardsOffSpecificDataPathAboveWatermark() throws Exception
 
         // there should be one shard on a bad path on node0
         execute(
-            "SELECT count(*) " +
+            "SELECT path " +
             "FROM sys.shards " +
             "WHERE table_name = 't' " +
             "  AND schema_name = 'doc'" +
-            "  AND node['id'] = ?" +
-            "  AND path LIKE concat(?, '%')",
```

Comment:
i did it with `like` originally, but it seems that it produces wrong regex patters on windows